src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-python.mdx

```diff
@@ -0,0 +1,818 @@
+
+```python
+##########
+# Traces #
+##########
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import OTLPSpanExporter
+from opentelemetry.trace.status import Status, StatusCode
+
+# Initialize tracing and an exporter that can send data to an OTLP endpoint
+# SELECT * FROM Span WHERE instrumentation.provider='opentelemetry'
+trace.set_tracer_provider(TracerProvider(resource=Resource.create(OTEL_RESOURCE_ATTRIBUTES)))
+trace.get_tracer_provider().add_span_processor(BatchSpanProcessor(OTLPSpanExporter()))
+```
+
+### E. Add instrumentation libraries: metrics [#library-metrics]
+
+In `app.py` add the following after the `Traces` section you added in Step D:
+
+```python
+###########
+# Metrics #
+###########
+from opentelemetry import metrics
+from opentelemetry.sdk.metrics import MeterProvider
+from opentelemetry.sdk.metrics.export import PeriodicExportingMetricReader
+from opentelemetry.exporter.otlp.proto.grpc.metric_exporter import OTLPMetricExporter
+
+# Initialize metering and an exporter that can send data to an OTLP endpoint
+# SELECT count(`http.server.active_requests`) FROM Metric FACET `service.name` TIMESERIES
+metrics.set_meter_provider(MeterProvider(resource=Resource.create(OTEL_RESOURCE_ATTRIBUTES), metric_readers=[PeriodicExportingMetricReader(OTLPMetricExporter())]))
+metrics.get_meter_provider()
+fib_counter = metrics.get_meter("opentelemetry.instrumentation.custom").create_counter("fibonacci.invocations", unit="1", description="Measures the number of times the fibonacci method is invoked.")
+```
+
+### F. Add instrumentation libraries: logs [#library-logs]
+
+In `app.py`, add the following after the `Metrics` section. This will import the logging module and set the `basicConfig` logging level to `DEBUG`:
+
+    ```python 
+    ########
+    # Logs # - OpenTelemetry Logs are still in the experimental state, so function names may change in the future
+    ########
+    import logging
+    logging.basicConfig(level=logging.DEBUG)
+
+    from opentelemetry import _logs
+    from opentelemetry.sdk._logs import LoggerProvider, LoggingHandler
+    from opentelemetry.sdk._logs.export import BatchLogRecordProcessor
+    from opentelemetry.exporter.otlp.proto.grpc._log_exporter import OTLPLogExporter
+
+    # Initialize logging and an exporter that can send data to an OTLP endpoint by attaching OTLP handler to root logger
+    # SELECT * FROM Log WHERE instrumentation.provider='opentelemetry'
+    _logs.set_logger_provider(LoggerProvider(resource=Resource.create(OTEL_RESOURCE_ATTRIBUTES)))
+    logging.getLogger().addHandler(LoggingHandler(logger_provider=_logs.get_logger_provider().add_log_record_processor(BatchLogRecordProcessor(OTLPLogExporter()))))
+    ```
+
+
+### G. Add Flask instrumentation [#flask-instrum]
```

Comment:
@homelessbirds Can you help me figure out what is wrong with the way this link is created? It looks the same as the other, but it's the only one that's showing up like this in the preview: 

<img width="504" alt="Screenshot 2023-06-22 at 2 07 43 PM" src="https://github.com/newrelic/docs-website/assets/48657837/0d705029-6108-4e11-a4f5-9b638be0295c">